### ISSUE TITLE
Does this project work?

### ISSUE THREAD

**Issue by Pzixel** [lukeautry/openapi-ts-client-gen#7]
I wanted to generate some clients for my TS code. Unfortunately I couldn't run it. When I run it I get

```
$ openapi-ts-client-gen
/c/Users/usr/AppData/Roaming/npm/openapi-ts-client-gen: line 1: /node_modules/openapi-ts-client-gen/dist/cli.js: No such file or directory
```

**Comment by skyl** [comment]
```
$ openapi-ts-client-gen

openapi-ts-client-gen: line 1: use strict: command not found
openapi-ts-client-gen: line 2: syntax error near unexpected token `('
openapi-ts-client-gen: line 2: `var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {'
```

**Comment by lukeautry** [comment]
Fixed in 1.1.1
